### ISSUE TITLE
Add Localized __content.css for Page-Level Overall Styling

### ISSUE THREAD

**Issue by hollowaykeanho** [ZORALab/Hestia#74]
# Description
```
Please provide a short description of what feature you're looking forward to
add into the existing repository below. Please include the story behind your
idea as well to give a better visualization of your idea.
```
It's quite difficult to ask users to consolidate all the pages' specific CSS stylesheets into `assets/` directory. We can offer `__content.css` as a direct source instead.

# Expected Behavior
```
Please specify the expected behavior of your requested feature. Some great and
helpful pointers are your expected interface (e.g. command patterns, simple
sketches of the user interface, etc).
```
`__content.css` will be loaded as the last CSS list item governing the page styling.

# Current Absent Behavior
```
Please specify the current behavior (e.g. workaround, blockage, etc).
```
`__content.css` is not responsive to `hestiaHUGO`.

# Attachment
```
Please drag and drop the necessary data files (e.g. screenshot, logs, etc)
below.
```

**Comment by hollowaykeanho** [comment]
Implemented in 4e5d11172fe63f2e9fba091f10a49e3cd58afb30

**Comment by hollowaykeanho** [comment]
reworded in b6c99f2391aea016b722f11d456a9414befdd9d7

**Comment by hollowaykeanho** [comment]
Released in `v1.1.0`.
